numpy/lib/tests/test_utils.py

```diff
@@ -51,6 +78,23 @@ def test_deprecate_fn():
     assert_('new_func3' in new_func3.__doc__)
 
 
+def test_deprecate_help_indentation():
+    assert_('    ' not in inspect.getdoc(old_func4))
+    assert_('    ' not in inspect.getdoc(old_func5))
```

Comment:
Wait, isn't this expected before "Bizarre indentation"?

Reply:
As far as I remember, we are trying to make the indentation in `help()` the same with and without deprecation. The output of help is processed by `cleandoc()`, which dedents all lines except the first one by as much as it can.

Reply:
Got it. Would be nice if the test could be more obviously comparing the deprecated and non-deprecated docstrings, but that's probably not worth the effort to change now.